src/BUILD

```diff
@@ -9,10 +9,24 @@ exports_files(
         "pickles/pt_model_10K.files/*",
         "model/*",
         "vocabulary/*",
+        "Senti4SD-fast.jar",
+        "dsm.bin",
+        "Senti4SD.model",
     ]),
     visibility = ["//visibility:public"],
 )
 
+#py_library(
```

Comment:
you want this commented out?

Reply:
for this PR, yes.. I'll add it in another PR